includes/admin/class-wp-job-manager-settings.php

```diff
@@ -153,26 +153,26 @@ protected function init_settings() {
 							'name'       => 'job_manager_hide_filled_positions',
 							'std'        => '0',
 							'label'      => __( 'Filled Positions', 'wp-job-manager' ),
-							'cb_label'   => __( 'Hide filled positions', 'wp-job-manager' ),
-							'desc'       => __( 'Filled positions will not display in your archives.', 'wp-job-manager' ),
+							'cb_label'   => __( 'Hide filled positions in job archives/search', 'wp-job-manager' ),
+							'desc'       => __( 'Filled positions will not display in your archives or in search.', 'wp-job-manager' ),
 							'type'       => 'checkbox',
 							'attributes' => [],
 						],
 						[
 							'name'       => 'job_manager_hide_expired',
 							'std'        => get_option( 'job_manager_hide_expired_content' ) ? '1' : '0', // back compat.
-							'label'      => __( 'Hide Expired Listings', 'wp-job-manager' ),
-							'cb_label'   => __( 'Hide expired listings in job archives/search', 'wp-job-manager' ),
-							'desc'       => __( 'Expired job listings will not be searchable.', 'wp-job-manager' ),
+							'label'      => __( 'Expired Positions', 'wp-job-manager' ),
+							'cb_label'   => __( 'Hide expired positions in job archives/search', 'wp-job-manager' ),
+							'desc'       => __( 'Expired positions will not display in your archives or in search.', 'wp-job-manager' ),
 							'type'       => 'checkbox',
 							'attributes' => [],
 						],
 						[
 							'name'       => 'job_manager_hide_expired_content',
 							'std'        => '1',
-							'label'      => __( 'Hide Expired Listings Content', 'wp-job-manager' ),
-							'cb_label'   => __( 'Hide content in expired single job listings', 'wp-job-manager' ),
-							'desc'       => __( 'Your site will display the titles of expired listings, but not the content of the listings. Otherwise, expired listings display their full content minus the application area.', 'wp-job-manager' ),
+							'label'      => __( 'Hide Expired Positions Content', 'wp-job-manager' ),
+							'cb_label'   => __( 'Hide content in expired single job position', 'wp-job-manager' ),
+							'desc'       => __( 'Your site will display the titles of expired position, but not the content of the positions Otherwise, expired positions display their full content minus the application area.', 'wp-job-manager' ),
```

Comment:
Some plural is wrong (I think) and there is a dot missing.

```suggestion
							'desc'       => __( 'Your site will display the titles of expired positions, but not the content of the positions. Otherwise, expired positions display their full content minus the application area.', 'wp-job-manager' ),
```

Reply:
Good catch, yep. Committing your suggested edits.